<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateItemDialogue</class>
 <widget class="QDialog" name="UpdateItemDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblProductName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProductNameDisplay">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblQuantity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbQuantity">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>281</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Image:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblImageDisplay">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>301</width>
     <height>301</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCofirmUpdate">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm and Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
